modules/social_features/social_user/src/Routing/RouteSubscriber.php

```diff
@@ -45,6 +45,21 @@ protected function alterRoutes(RouteCollection $collection) {
       $route->setOption('_admin_route', FALSE);
     }
 
+    // Routes for which needs to disable access if the user is blocked.
+    $disable_access_for = [
+      'view.groups.page_user_groups',
+      'view.user_information.user_information',
+      'view.user_discussion.page_user_discussions',
```

Comment:
@dmytriikaiun Since discussions are an external module, and this route doesn't exist in the clean distro, so we usually recommended to remove such lines of code from here and fix it in the related module(s).
But I think this is not so critical so please create follow up to fix it in other modules (Discussions, Challenges, Courses)